### PR TITLE
Add some locales to controls

### PR DESCRIPTION
**Pull request by dtrucs**

### CODE REVIEW

File: src/modules/Map/Map/Map.js [hunk 1]
```diff
@@ -457,9 +473,9 @@ export class MapComponent extends React.Component {
           const { _container: container } = controlInstance;
 
           if (control === CONTROL_NAVIGATION) {
-            container.querySelector('.mapboxgl-ctrl-zoom-in').setAttribute('title', translate('terralego.map.zoom_in_control.title'));
-            container.querySelector('.mapboxgl-ctrl-zoom-out').setAttribute('title', translate('terralego.map.zoom_out_control.title'));
-            container.querySelector('.mapboxgl-ctrl-compass-arrow').setAttribute('title', translate('terralego.map.compass_arrow_control.title'));
+            ['zoom-in', 'zoom-out', 'compass-arrow'].forEach(value => {
+              container.querySelector(`.mapboxgl-ctrl-${value}`).setAttribute('title', translate(`terralego.map.${value.replace(/-/, '_')}_control.title`));
+            });
```
Comment (mabhub) [inline src/modules/Map/Map/Map.js]:
```jsx
            ['zoom-in', 'zoom-out', 'compass-arrow'].forEach(controlName => {
              const domElement = container.querySelector(`.mapboxgl-ctrl-${controlName}`);

              if (domElement) {
                const translationKey = `terralego.map.${controlName.replace(/-/, '_')}_control.title`;
                domElement.setAttribute('title', translate(translationKey));
              }
            }
```

File: src/modules/Map/Map/Map.js [hunk 2]
```diff
@@ -408,6 +414,16 @@ export class MapComponent extends React.Component {
...
+
+          const { _container: container } = map;
+
+          ['point', 'polygon', 'line', 'trash', 'combine', 'uncombine'].forEach(value => {
+            const element = container.querySelector(`.mapbox-gl-draw_${value}`);
+            if (element === null) {
+              return;
+            }
+            element.setAttribute('title', translate(`terralego.map.draw_control.${value}`));
+          });
```
Comment (mabhub) [inline src/modules/Map/Map/Map.js]:
```jsx
          ['point', 'polygon', 'line', 'trash', 'combine', 'uncombine'].forEach(actionName => {
            const element = container.querySelector(`.mapbox-gl-draw_${actionName}`);

            if (element) {
              element.setAttribute('title', translate(`terralego.map.draw_control.${actionName}`));
            }
          }); 
```

Comment (SebCorbin) [reply src/modules/Map/Map/Map.js]: This won't get picked up by i18n extract

Comment (mabhub) [reply src/modules/Map/Map/Map.js]: Original proposed version would?

Comment (SebCorbin) [reply src/modules/Map/Map/Map.js]: Nope, I'm hacking this conversation to say that don't ever use a variable inside a translation key name or it won't be picked up by automatic extraction

Comment (mabhub) [reply src/modules/Map/Map/Map.js]: Any suggestion of improvement? 😇

Comment (dtrucs) [reply src/modules/Map/Map/Map.js]: @SebCorbin I don't get why?
I tried on TA linked to TF on this branch with translations set and it's ok. 
image
Do you know an edge case?

Comment (SebCorbin) [reply src/modules/Map/Map/Map.js]: @dtrucs translations works, but it won't extract these keys to the `locales/__X__/translation.json` files. So :
- either you put them manually in `translation.json` file (with a chance of not being up to date if modified)
- either you call translate(`terralego.map.draw_control.xxxx`) for each component
- either you follow some guidelines like use of context https://i18next-extract.netlify.com/#/comment-hints?id=explicitly-specify-contexts-for-a-key

Comment (dtrucs) [reply src/modules/Map/Map/Map.js]: Okay i changed it 👍

File: src/modules/Map/Map/Map.js [hunk 3]
```diff
@@ -408,6 +414,23 @@ export class MapComponent extends React.Component {
...
+          const { _container: container } = map;
+          const drawControlLocales = {
+            point: translate('terralego.map.draw_control.point'),
+            polygon: translate('terralego.map.draw_control.polygon'),
+            line: translate('terralego.map.draw_control.line'),
+            trash: translate('terralego.map.draw_control.trash'),
+            combine: translate('terralego.map.draw_control.combine'),
+            uncombine: translate('terralego.map.draw_control.uncombine'),
+          };
+          Object.entries(drawControlLocales).forEach(([drawControl, locale]) => {
```
Comment (mabhub) [inline src/modules/Map/Map/Map.js]:
```suggestion
          Object.entries(drawControlLocales).forEach(([drawControl, translation]) => {
```

File: src/modules/Map/Map/Map.js [hunk 4]
```diff
@@ -408,6 +414,23 @@ export class MapComponent extends React.Component {
...
+            const domElement = container.querySelector(`.mapbox-gl-draw_${drawControl}`);
+
+            if (domElement) {
+              domElement.setAttribute('title', locale);
```
Comment (mabhub) [inline src/modules/Map/Map/Map.js]:
```suggestion
              domElement.setAttribute('title', translation);
```

**Pull request merged**